homeassistant/components/binary_sensor/nest.py

```diff
@@ -60,11 +61,8 @@
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Setup Nest binary sensors."""
-    if discovery_info is None:
```

Comment:
This is actually needed. Without this, the configurator never shows up https://github.com/home-assistant/home-assistant/issues/4730